pkg/libcontainer/libcontainer_linux.go

```diff
@@ -0,0 +1,104 @@
+//go:build linux
+// +build linux
+
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package libcontainer
+
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import (
+	libcontainercgroups "github.com/opencontainers/runc/libcontainer/cgroups"
+	libcontainerfscommon "github.com/opencontainers/runc/libcontainer/cgroups/fscommon"
+	libcontainermanager "github.com/opencontainers/runc/libcontainer/cgroups/manager"
+	libcontainersystemd "github.com/opencontainers/runc/libcontainer/cgroups/systemd"
+	libcontainerconfigs "github.com/opencontainers/runc/libcontainer/configs"
+	libcontainerutils "github.com/opencontainers/runc/libcontainer/utils"
+)
+
+var (
```

Comment:
is it problematic these can be mutated? should we do `func X() { return libcontainercgroups.X() }` instead?